Makefile

```diff
@@ -1,3 +1,3 @@
-all    :; dapp --use solc:0.6.12 build
-clean  :; dapp clean
+all    :; forge build
```

Comment:
I think you still need to force the version here otherwise will use whatever is your latest in your local `forge`